gazebo_ros_control/src/default_robot_hw_sim.cpp

```diff
@@ -177,7 +177,7 @@ bool DefaultRobotHWSim::initSim(
     else
     {
       ROS_FATAL_STREAM_NAMED("default_robot_hw_sim","No matching hardware interface found for '"
-        << hardware_interface );
+        << hardware_interface << " while loading interfaces for " << joint_names_[j] );
```

Comment:
do you mind fixing the missing single quote ``'`` around hardware_interface while you are at it?